diff -ru file-5.45.orig/magic/Magdir/images file-5.45/magic/Magdir/images
--- file-5.45.orig/magic/Magdir/images	2023-07-27 20:04:45.000000000 +0200
+++ file-5.45/magic/Magdir/images	2024-09-09 11:02:19.317463825 +0200
@@ -3610,7 +3610,7 @@
 
 # Microsoft Paint graphic
 # http://www.fileformat.info/format/mspaint/egff.htm
-0	string	DanM 	icrosoft Paint image data (version 1.x)
+0	string	DanM 	Microsoft Paint image data (version 1.x)
 >4	uleshort	x	(%d
 >>6	uleshort	x	x %d)
 0	string	LinS 	Microsoft Paint image data (version 2.0)
