diff -ru gettext-0.25.orig/gettext-runtime/gnulib-lib/getlocalename_l-unsafe.c gettext-0.25/gettext-runtime/gnulib-lib/getlocalename_l-unsafe.c
--- gettext-0.25.orig/gettext-runtime/gnulib-lib/getlocalename_l-unsafe.c	2025-05-04 15:43:18.000000000 +0200
+++ gettext-0.25/gettext-runtime/gnulib-lib/getlocalename_l-unsafe.c	2025-06-24 19:28:41.974408152 +0200
@@ -34,7 +34,7 @@
 #endif
 #include "setlocale_null.h"
 
-#if (__GLIBC__ >= 2 && !defined __UCLIBC__) || (defined __linux__ && HAVE_LANGINFO_H) || defined __CYGWIN__
+#if (__GLIBC__ >= 2 && !defined __UCLIBC__) || (defined __linux__ && HAVE_LANGINFO_H) || defined __CYGWIN__ || defined __midipix__
 # include <langinfo.h>
 #endif
 #if defined __sun
@@ -499,7 +499,7 @@
            nl_langinfo_l (_NL_LOCALE_NAME (category), locale).  */
         name = locale->__names[category];
       return (struct string_with_storage) { name, STORAGE_OBJECT };
-#elif defined __linux__ && HAVE_LANGINFO_H && defined NL_LOCALE_NAME
+#elif defined __linux__ && HAVE_LANGINFO_H && defined NL_LOCALE_NAME || defined __midipix__
       /* musl libc */
       const char *name = nl_langinfo_l (NL_LOCALE_NAME (category), locale);
       return (struct string_with_storage) { name, STORAGE_OBJECT };
diff -ru gettext-0.25.orig/gettext-runtime/intl/gnulib-lib/getlocalename_l-unsafe.c gettext-0.25/gettext-runtime/intl/gnulib-lib/getlocalename_l-unsafe.c
--- gettext-0.25.orig/gettext-runtime/intl/gnulib-lib/getlocalename_l-unsafe.c	2025-05-04 15:43:19.000000000 +0200
+++ gettext-0.25/gettext-runtime/intl/gnulib-lib/getlocalename_l-unsafe.c	2025-06-24 19:28:04.198562237 +0200
@@ -34,7 +34,7 @@
 #endif
 #include "setlocale_null.h"
 
-#if (__GLIBC__ >= 2 && !defined __UCLIBC__) || (defined __linux__ && HAVE_LANGINFO_H) || defined __CYGWIN__
+#if (__GLIBC__ >= 2 && !defined __UCLIBC__) || (defined __linux__ && HAVE_LANGINFO_H) || defined __CYGWIN__ || defined __midipix__
 # include <langinfo.h>
 #endif
 #if defined __sun
@@ -499,7 +499,7 @@
            nl_langinfo_l (_NL_LOCALE_NAME (category), locale).  */
         name = locale->__names[category];
       return (struct string_with_storage) { name, STORAGE_OBJECT };
-#elif defined __linux__ && HAVE_LANGINFO_H && defined NL_LOCALE_NAME
+#elif defined __linux__ && HAVE_LANGINFO_H && defined NL_LOCALE_NAME || defined __midipix__
       /* musl libc */
       const char *name = nl_langinfo_l (NL_LOCALE_NAME (category), locale);
       return (struct string_with_storage) { name, STORAGE_OBJECT };
diff -ru gettext-0.25.orig/gettext-tools/gnulib-lib/getlocalename_l-unsafe.c gettext-0.25/gettext-tools/gnulib-lib/getlocalename_l-unsafe.c
--- gettext-0.25.orig/gettext-tools/gnulib-lib/getlocalename_l-unsafe.c	2025-05-04 15:43:20.000000000 +0200
+++ gettext-0.25/gettext-tools/gnulib-lib/getlocalename_l-unsafe.c	2025-06-24 19:29:07.130896346 +0200
@@ -34,7 +34,7 @@
 #endif
 #include "setlocale_null.h"
 
-#if (__GLIBC__ >= 2 && !defined __UCLIBC__) || (defined __linux__ && HAVE_LANGINFO_H) || defined __CYGWIN__
+#if (__GLIBC__ >= 2 && !defined __UCLIBC__) || (defined __linux__ && HAVE_LANGINFO_H) || defined __CYGWIN__ || defined __midipix__
 # include <langinfo.h>
 #endif
 #if defined __sun
@@ -499,7 +499,7 @@
            nl_langinfo_l (_NL_LOCALE_NAME (category), locale).  */
         name = locale->__names[category];
       return (struct string_with_storage) { name, STORAGE_OBJECT };
-#elif defined __linux__ && HAVE_LANGINFO_H && defined NL_LOCALE_NAME
+#elif defined __linux__ && HAVE_LANGINFO_H && defined NL_LOCALE_NAME || defined __midipix__
       /* musl libc */
       const char *name = nl_langinfo_l (NL_LOCALE_NAME (category), locale);
       return (struct string_with_storage) { name, STORAGE_OBJECT };
