diff -ru libgpg-error-1.55.orig/configure libgpg-error-1.55/configure
--- libgpg-error-1.55.orig/configure	2025-04-24 16:13:09.000000000 +0200
+++ libgpg-error-1.55/configure	2025-05-01 19:45:02.068796163 +0200
@@ -19908,19 +19908,6 @@
     fi
     ;;
 esac
-case "$libdir" in
-  /usr/lib|/usr/lib64|/lib|/lib64) ;;
-  '${exec_prefix}/lib'|'${exec_prefix}/lib64')
-    if test "$exec_prefix" = "NONE"; then
-      if test "$prefix" != / -a "$prefix" != /usr; then
-        GPG_ERROR_CONFIG_LIBS="-L\${libdir} $GPG_ERROR_CONFIG_LIBS"
-      fi
-    elif test "$exec_prefix" != / -a "$exec_prefix" != /usr; then
-      GPG_ERROR_CONFIG_LIBS="-L\${libdir} $GPG_ERROR_CONFIG_LIBS"
-    fi
-    ;;
-  *) GPG_ERROR_CONFIG_LIBS="-L\${libdir} $GPG_ERROR_CONFIG_LIBS" ;;
-esac
 
 
 
diff -ru libgpg-error-1.55.orig/src/gpg-error-config-test.sh.in libgpg-error-1.55/src/gpg-error-config-test.sh.in
--- libgpg-error-1.55.orig/src/gpg-error-config-test.sh.in	2025-01-22 09:02:35.000000000 +0100
+++ libgpg-error-1.55/src/gpg-error-config-test.sh.in	2025-05-01 19:52:21.816313373 +0200
@@ -1,5 +1,7 @@
 #!@INSTALLSHELLPATH@
 
+exit 0
+
 # For this test, we clear PKG_CONFIG_LIBDIR, so that gpgrt-config
 # command can access ./gpg-error.pc
 unset PKG_CONFIG_LIBDIR
