diff -ru libgpg-error-1.55.orig/configure libgpg-error-1.55/configure
--- libgpg-error-1.55.orig/configure	2025-04-24 16:13:09.000000000 +0200
+++ libgpg-error-1.55/configure	2025-05-01 12:58:03.282492116 +0200
@@ -19908,19 +19908,6 @@
     fi
     ;;
 esac
-case "$libdir" in
-  /usr/lib|/usr/lib64|/lib|/lib64) ;;
-  '${exec_prefix}/lib'|'${exec_prefix}/lib64')
-    if test "$exec_prefix" = "NONE"; then
-      if test "$prefix" != / -a "$prefix" != /usr; then
-        GPG_ERROR_CONFIG_LIBS="-L\${libdir} $GPG_ERROR_CONFIG_LIBS"
-      fi
-    elif test "$exec_prefix" != / -a "$exec_prefix" != /usr; then
-      GPG_ERROR_CONFIG_LIBS="-L\${libdir} $GPG_ERROR_CONFIG_LIBS"
-    fi
-    ;;
-  *) GPG_ERROR_CONFIG_LIBS="-L\${libdir} $GPG_ERROR_CONFIG_LIBS" ;;
-esac
 
 
 
diff -ru libgpg-error-1.55.orig/src/Makefile.in libgpg-error-1.55/src/Makefile.in
--- libgpg-error-1.55.orig/src/Makefile.in	2025-04-24 16:13:09.000000000 +0200
+++ libgpg-error-1.55/src/Makefile.in	2025-05-01 13:02:33.363492167 +0200
@@ -575,7 +575,7 @@
 nodist_include_HEADERS = gpg-error.h gpgrt.h
 @INSTALL_GPG_ERROR_CONFIG_FALSE@bin_SCRIPTS = gpgrt-config
 @INSTALL_GPG_ERROR_CONFIG_TRUE@bin_SCRIPTS = gpgrt-config gpg-error-config
-@INSTALL_GPG_ERROR_CONFIG_FALSE@noinst_SCRIPTS = gpg-error-config
+@INSTALL_GPG_ERROR_CONFIG_FALSE@noinst_SCRIPTS =
 m4datadir = $(datadir)/aclocal
 m4data_DATA = gpg-error.m4 gpgrt.m4
 pkgconfigdir = $(libdir)/pkgconfig
