diff -ru lighttpd-1.4.77.orig/src/mod_openssl.c lighttpd-1.4.77/src/mod_openssl.c
--- lighttpd-1.4.77.orig/src/mod_openssl.c	2025-01-10 05:12:08.000000000 +0100
+++ lighttpd-1.4.77/src/mod_openssl.c	2025-02-22 13:20:20.502779996 +0100
@@ -3020,15 +3020,7 @@
         "X25519:P-256:P-384:X448";
        #endif
 
-   #if (defined(BORINGSSL_API_VERSION) && BORINGSSL_API_VERSION >= 19) \
-    || (defined(LIBRESSL_VERSION_NUMBER) \
-        && LIBRESSL_VERSION_NUMBER >= 0x2050100fL) \
-    || OPENSSL_VERSION_NUMBER >= 0x10101000L
     int rc = SSL_CTX_set1_groups_list(s->ssl_ctx, groups);
-   #elif (defined(BORINGSSL_API_VERSION && BORINGSSL_API_VERSION >= 3) \
-      || OPENSSL_VERSION_NUMBER >= 0x10100000L
-    int rc = SSL_CTX_set1_curves_list(s->ssl_ctx, groups);
-   #endif
     if (1 != rc) {
         log_error(srv->errh, __FILE__, __LINE__,
           "SSL: Unable to config groups %s", groups);
