diff -ru posh-master.orig/configure.ac posh-master/configure.ac
--- posh-master.orig/configure.ac	2025-09-22 17:53:37.765801006 +0200
+++ posh-master/configure.ac	2025-09-22 17:55:30.306251926 +0200
@@ -2,7 +2,7 @@
 dnl Process this file with autoconf to produce a configure script
 dnl
 AC_INIT([Debian posh],[POSH_VERSION],[clint@debian.org],[posh])
-AC_PREREQ([2.72])
+AC_PREREQ([2.69])
 AM_INIT_AUTOMAKE([subdir-objects])
 AC_CONFIG_HEADERS([config.h])
 AC_USE_SYSTEM_EXTENSIONS
