diff -ru thttpd-2.29.orig/cgi-src/Makefile.in thttpd-2.29/cgi-src/Makefile.in
--- thttpd-2.29.orig/cgi-src/Makefile.in	2005-05-20 21:10:25.000000000 +0200
+++ thttpd-2.29/cgi-src/Makefile.in	2025-09-25 21:12:17.207014505 +0200
@@ -35,7 +35,7 @@
 DEFS =		@DEFS@
 INCLS =		-I..
 CFLAGS =	$(CCOPT) $(DEFS) $(INCLS)
-LDFLAGS =	@LDFLAGS@ @V_STATICFLAG@
+LDFLAGS =	@LDFLAGS@
 LIBS =		@LIBS@
 NETLIBS =	@V_NETLIBS@
 INSTALL =	@INSTALL@
@@ -51,15 +51,15 @@
 all:		redirect ssi phf
 
 redirect:	redirect.o
-	$(CC) $(LDFLAGS) $(STATICFLAG) redirect.o $(LIBS) -o redirect
+	$(CC) $(LDFLAGS) redirect.o $(LIBS) -o redirect
 
 ssi:		ssi.o ../match.o
-	$(CC) $(LDFLAGS) $(STATICFLAG) ssi.o ../match.o $(LIBS) -o ssi
+	$(CC) $(LDFLAGS) ssi.o ../match.o $(LIBS) -o ssi
 
 ssi.o:		../match.h
 
 phf:		phf.o
-	$(CC) $(LDFLAGS) $(STATICFLAG) phf.o $(LIBS) -o phf
+	$(CC) $(LDFLAGS) phf.o $(LIBS) -o phf
 
 strerror.o:
 	@rm -f strerror.o
diff -ru thttpd-2.29.orig/extras/Makefile.in thttpd-2.29/extras/Makefile.in
--- thttpd-2.29.orig/extras/Makefile.in	2005-05-20 21:10:25.000000000 +0200
+++ thttpd-2.29/extras/Makefile.in	2025-09-25 21:10:07.441859846 +0200
@@ -36,7 +36,7 @@
 DEFS =		@DEFS@
 INCLS =		-I..
 CFLAGS =	$(CCOPT) $(DEFS) $(INCLS)
-STATICFLAG =	@V_STATICFLAG@
+STATICFLAG =
 LDFLAGS =	@LDFLAGS@
 LIBS =		@LIBS@
 NETLIBS =	@V_NETLIBS@
@@ -68,7 +68,6 @@
 install:	all
 	rm -f $(BINDIR)/makeweb $(BINDIR)/htpasswd $(BINDIR)/syslogtocern
 	cp makeweb $(BINDIR)/makeweb
-	chgrp $(WEBGROUP) $(BINDIR)/makeweb
 	chmod 2755 $(BINDIR)/makeweb
 	cp htpasswd $(BINDIR)/htpasswd
 	cp syslogtocern $(BINDIR)/syslogtocern
diff -ru thttpd-2.29.orig/Makefile.in thttpd-2.29/Makefile.in
--- thttpd-2.29.orig/Makefile.in	2005-05-20 21:10:24.000000000 +0200
+++ thttpd-2.29/Makefile.in	2025-09-25 21:10:07.441980194 +0200
@@ -110,11 +110,11 @@
 
 installthis:
 	-mkdir -p $(DESTDIR)$(BINDIR)
-	$(INSTALL) -m 555 -o bin -g bin thttpd $(DESTDIR)$(BINDIR)
+	$(INSTALL) -m 755 thttpd $(DESTDIR)$(BINDIR)
 
 install-man:
 	-mkdir -p $(DESTDIR)$(MANDIR)/man8
-	$(INSTALL) -m 444 -o bin -g bin thttpd.8 $(DESTDIR)$(MANDIR)/man8
+	$(INSTALL) -m 444 thttpd.8 $(DESTDIR)$(MANDIR)/man8
 
 installsubdirs:
 	for i in $(SUBDIRS) ; do ( \
